lib/journaled/audit_log.rb

```diff
@@ -84,19 +104,16 @@ def enabled?
     end
 
     class_methods do
-      def has_audit_log(ignore: [])
-        ignored_columns = _audit_log_inherited_ignored_columns + [ignore].flatten(1)
-        self.audit_log_config = Config.new(true, ignored_columns.uniq)
+      def has_audit_log(ignore: [], enqueue_with: {})
+        self.audit_log_config = audit_log_config.dup
```

Comment:
Relying on implicit inheritance of `class_attribute` and a deep `.dup` here, instead of explicitly handing off the inherited "ignore" and "enqueue_with" configs separately.